book/quick-start/mock.md

```diff
@@ -136,6 +176,7 @@ Now, you should have the following in your `.env` file:
 | `PRIVATE_KEY` | Private key for the account that will be deploying the contract and relaying proofs on-chain. |
 | `ETHERSCAN_API_KEY` | Etherscan API key for verifying the deployed contracts. |
 | `L2OO_ADDRESS` | The address of the `OPSuccinctL2OutputOracle` contract from the previous step. |
+| `DGF_ADDRESS` | *Optional* The address of the `OPSuccinctDisputeGameFactory` contract from the previous step. |
```

Comment:
The user should only configure `L2OO_ADDRESS` OR the `DGF_ADDRESS` right?

Reply:
`DGF_ADDRESS` seems to take precedence in the proposer.

Reply:
Yes and L2OO_ADDRESS is needed to deploy the DisputeGameFactory